app/controllers/subscribers_controller.rb

```diff
@@ -9,17 +9,30 @@ def subscriptions
     render json: { subscriber: subscriber.as_json, subscriptions: subscriptions }
   end
 
+  def change_address
+    subscriber.address = new_address
+    if subscriber.save
+      render json: { subscriber: subscriber }
+    else
+      render json: { message: subscriber.errors.full_messages.to_sentence }, status: 422
```

Comment:
Can we do `subscriber.update(address: new_address)` and leave the validation exception to show this message?

Reply:
Done!